java/src/jmri/jmrit/logix/Engineer.java

```diff
@@ -112,9 +114,9 @@ public void run() {
                         break;
                     }
                 } catch (InterruptedException ie) {
-                    log.error("InterruptedException " + ie);
+                    log.error("Engineer Interrupted at time" + ie);
```

Comment:
Do you want the stack trace here?

If so, this needs to be ```...time", ie);```

If you don't want the stack trace, why include the exception class in the error message?